specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -3384,51 +3384,64 @@
         "startDate"
       ]
     },
-    "Filters": {
+    "BudgetFilter": {
       "description": "May be used to filter budgets by resource group, resource, or meter.",
       "properties": {
-        "resourceGroups": {
-          "description": "The list of filters on resource groups, allowed at subscription level only.",
+        "and": {
+          "description": "The logical \"AND\" expression. Must have at least 2 items.",
           "type": "array",
           "items": {
-            "type": "string"
+            "$ref": "#/definitions/BudgetFilter"
           },
-          "minItems": 0,
-          "maxItems": 10
+          "minItems": 2
         },
-        "resources": {
-          "description": "The list of filters on resources.",
-          "type": "array",
-          "items": {
-            "type": "string"
-          },
-          "minItems": 0,
-          "maxItems": 10
+        "not": {
+          "description": "The logical \"NOT\" expression.",
+          "$ref": "#/definitions/BudgetFilter"
         },
-        "meters": {
-          "description": "The list of filters on meters (GUID), mandatory for budgets of usage category. ",
-          "type": "array",
-          "items": {
-            "type": "string",
-            "format": "uuid"
-          },
-          "minItems": 0,
-          "maxItems": 10
+        "dimensions": {
+          "description": "Has comparison expression for a dimension",
+          "$ref": "#/definitions/BudgetComparisonExpression"
         },
         "tags": {
-          "type": "object",
-          "description": "The dictionary of filters on tags.",
-          "additionalProperties": {
-            "type": "array",
-            "items": {
-              "type": "string"
-            }
-          },
-          "minItems": 0,
-          "maxItems": 10
+          "description": "Has comparison expression for a tag",
+          "$ref": "#/definitions/BudgetComparisonExpression"
         }
       }
     },
+    "BudgetComparisonExpression": {
+      "description": "The comparison expression to be used in the budgets.",
+      "properties": {
+        "name": {
+          "description": "The name of the column to use in comparison.",
+          "type": "string"
+        },
+        "operator": {
+          "description": "The operator to use for comparison.",
+          "type": "string",
+          "enum": [
+            "In"
+          ],
+          "x-ms-enum": {
+            "name": "OperatorType",
```

Comment:
Please check the x-ms-enum name, it conflicts with this one https://github.com/Azure/azure-rest-api-specs/blob/e8fd0dcdc31abae062b809741cffc0a9ac9edf65/specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json#L3471-L3479

Reply:
thanks for catching that. I've fixed it in the newest commit